config.h

```diff
@@ -60,6 +60,8 @@ typedef enum {
 	PV_CONTROL_REMOTE_ALWAYS,
 	PV_DEBUG_SHELL,
 	PV_DEBUG_SHELL_AUTOLOGIN,
+	PV_DEBUG_SHELL_ACTIVE,
```

Comment:
How is PV_DEBUG_SHELL different from PV_DEBUG_SHELL_ACTIVE. Is it configuring the same shell? is PV_DEBUG_SHELL deprecated now? If possible, please, remove PV_DEBUG_SHELL_ACTIVE and keep PV_DEBUG_SHELL working so we can be backwards compatible with previous configs.